ll-property-image.html

```diff
@@ -427,23 +427,23 @@ <h1 ondragstart="return false;">
       });
     },
     /**
-     * Toggles the Default Image value, and raises an event.
+     * Toggles the Primary Image value, and raises an event.
      */
-    makeDefaultImage: function() {
+    makePrimaryImage: function() {
       if(this.readOnly) {
         return;
       }
 
       var image = {
         imgId: this.imgId,
         name: this.name,
-        isDefault: true,
+        isPrimary: true,
         title: this.title,
         description: this.description,
         sortOrder: this.sortOrder,
         tags: this.tags
       };
-      this.fire('ll-property-image-default', image);
+      this.fire('ll-property-image-primary', image);
```

Comment:
Was anyone else listening to this event? Ih-app listening to `ll-property-image-default` ??